### PR TITLE
1 words in reverse

### LINKED ISSUE #3: 1 - Words in reverse
Generate 10 random words. Print each out in reverse then print out the whole sentence in reverse.

Hints
1. Generate a random sentence, and output the sentence in reverse.
    E.g. `I am a boy` becomes `yob a ma I`.
2. Generate a random sentence, and output the reverse of each word of the sentence.
    E.g. `I am a boy` becomes `I ma a yob`.

### PR DESCRIPTION
**Pull request by ebosetalee**
Resolves #3 [https://repl.it/join/adjguglq-emmanuellaesezo](https://repl.it/join/adjguglq-emmanuellaesezol)

### CODE REVIEW

**Comment by jirevwe** [comment]
This link is invalid, share another one

File: words-in-reverse.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+words = ["When", "life", "is",\
```
Comment (jirevwe) [inline words-in-reverse.py]: Why is this an array? It should be a string.

**Review by jirevwe — CHANGES_REQUESTED** [review]

**Comment by jirevwe** [comment]
Also, can you update it to request input from the user?

**Comment by ebosetalee** [comment]
A new link[https://repl.it/join/eeeulpmp-emmanuellaesezo](https://repl.it/join/eeeulpmp-emmanuellaesezo)

**Comment by jirevwe** [comment]
Your solution doesn't handle empty strings. 👀

**Review by jirevwe — APPROVED** [review]

**Comment by ebosetalee** [comment]
Yayyyyyyy!!!!!!!!
Thank you sir

**Comment by ebosetalee** [comment]
A new link [https://repl.it/join/eeeulpmp-emmanuellaesezo](https://repl.it/join/eeeulpmp-emmanuellaesezo)

File: words-in-reverse.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+words = input("Write a sentence containing 10 words (be creative): ")
+length = 10
+newwords = words.split(" ")
```
Comment (yudori) [inline words-in-reverse.py]: @ebosetalee The link I shared with you yesterday was to get you familiar with python naming conventions. It's good to start developing that culture of abiding to PEP-8 standards early as it helps unify ways of writing code and makes others understand your code more easily when they come across it.
We typically place an underscore between words in a variable, so this should be `new_words`.

I'll start requesting for changes from tomorrow if you don't follow it.

File: words-in-reverse.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+while True:
+    if length == len(newwords):
+        print(words[::-1])
+
+        print ("\n AND\n")
+
+        # newwords = words.split(" ")
+        sentence = [word[::-1] for word in newwords]
+        newSentence = " ".join(sentence)
```
Comment (yudori) [inline words-in-reverse.py]: `new_sentence`

File: words-in-reverse.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+        print(newSentence)
+        break
+    else:
+        break
```
Comment (yudori) [inline words-in-reverse.py]: The `while` loop isn't needed. The `else` block too. This should be sufficient.

```suggestion
    if length == len(newwords):
        print(words[::-1])

        print ("\n AND\n")

        # newwords = words.split(" ")
        sentence = [word[::-1] for word in newwords]
        newSentence = " ".join(sentence)
        print(newSentence)
```

**Review by yudori — CHANGES_REQUESTED** [review]
@ebosetalee , Good work. See comments below.

Also You're re-committing the changes from yesterday. Please remove it from your commit. Reach out if you need help with that.

**Comment by ebosetalee** [comment]
Solved @yudori

File: words-in-reverse.py [hunk 5]
```diff
@@ -1,16 +1,12 @@
...
-        newSentence = " ".join(sentence)
-        print(newSentence)
-        break
-    else:
-        break
\ No newline at end of file
+    print ("\n AND\n")
+
+    sentence = [word[::-1] for word in new_words]
+    new_Sentence = " ".join(sentence)
```
Comment (yudori) [inline words-in-reverse.py]: `new_sentence`

Comment (ebosetalee) [reply words-in-reverse.py]: Done

**Review by yudori — APPROVED** [review]
LGTM!

**Pull request merged**